### PR TITLE
Added in tests that check various values on the Kinto server

### PR DESCRIPTION
**Pull request by chartjes**

### CODE REVIEW

File: config-tests/test_server.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+def conf():
+    config = configparser.ConfigParser()
+    config.read('manifest.ini')
+    return config
+
+
+def test_heartbeat_fields(env, conf):
+    response = requests.get(conf.get(env, 'we_server_url') + '/__heartbeat__')
+    data = response.json()
+    fields = {'storage', 'portier', 'cache', 'attachments', 'permission', 'oauth'}
```
Comment (Natim) [inline config-tests/test_server.py]: The list of fields depends on the capability configured for the server.
For instance kinto-webextension will have oauth but not kinto-settings, also at Mozilla no stacks currently uses portier.

Comment (chartjes) [reply config-tests/test_server.py]: Yeah, I based this on what kinto-webextension is currently returning. This test is specific to kinto-webextensions

Comment (Natim) [reply config-tests/test_server.py]: See:
- [`https://webextensions-settings.stage.mozaws.net/v1/__heartbeat__`](https://webextensions-settings.stage.mozaws.net/v1/__heartbeat__)
- [`https://kinto.stage.mozaws.net/v1/__heartbeat__`](https://kinto.stage.mozaws.net/v1/__heartbeat__)
- [`https://kinto-writer.stage.mozaws.net/v1/__heartbeat__`](https://kinto-writer.stage.mozaws.net/v1/__heartbeat__)

Comment (Natim) [reply config-tests/test_server.py]: If you only target the webextension stack, you should not have portier nor attachment in there or it's a bug :)

**Pull request merged**